kubernetes/src/main/resources/kubernetes/kubernetes.bom

```diff
@@ -697,7 +738,9 @@ brooklyn.catalog:
             --address=0.0.0.0 \
             --allow-privileged=true \
             --config=/etc/kubernetes/manifests \
-            --hostname-override=${HOST_SUBNET_ADDRESS} \
```

Comment:
The removal of `hostname-override` is not in https://github.com/brooklyncentral/clocker/pull/329. Can you say why it's removed here?